lib/WelcomeScreen.dart

```diff
@@ -0,0 +1,221 @@
+import 'dart:js';
+import 'dart:ui';
+
+import 'package:flutter/cupertino.dart';
```

Comment:
Please remove this import as it is already included in the material.dart import by default.